scrapers_next/sd/committees.py

```diff
@@ -0,0 +1,60 @@
+from spatula import JsonListPage, JsonPage, SkipItem, URL
+from openstates.models import ScrapeCommittee
+
+
+class CommitteeList(JsonListPage):
+
+    source_string = "https://sdlegislature.gov/api/SessionCommittees/Session/68"
+    source = URL(source_string)
+
+    # I don't know if there's a method on the base class for this to be overridden
+    def standardize_chamber(self, original_chamber_text):
+        match original_chamber_text:
```

Comment:
Unfortunately, openstates-scrapers is still running on python 3.9, so this function isn't _quite_ there yet.

Reply:
Completed